pandas/core/strings.py

```diff
@@ -898,23 +898,76 @@ def str_join(arr, sep):
 
 def str_findall(arr, pat, flags=0):
     """
-    Find all occurrences of pattern or regular expression in the
-    Series/Index. Equivalent to :func:`re.findall`.
+    Find all occurrences of pattern or regular expression in the Series/Index.
+
+    Equivalent to apply :func:`re.findall` to all the elements in the
+    Series/Index.
 
     Parameters
     ----------
-    pat : string
-        Pattern or regular expression
-    flags : int, default 0 (no flags)
-        re module flags, e.g. re.IGNORECASE
+    pat : str
```

Comment:
Seeing this change now, I personally would prefer keeping it like 'string' (it's more pronouncable), but I suppose in the guidelines it is now 'str'. Something to discuss there .. :)